editions_test.go

```diff
@@ -53,6 +53,18 @@ func TestGetEdition(t *testing.T) {
 	}
 }
 
+func TestGetEditionISBN(t *testing.T) {
```

Comment:
Add another test case for when isbn keys don't have a length of 10 or 13 as well as a test case for when the openlibrary throws a 404.

Reply:
A quick note, you may want to use `t.Run(testname, function{})` for sub-testing which would make the output of the tests more readable.

For example,
TestGetEditionISBN would have a sub-test that is named Test Valid ISBN